src/main/java/org/example/mappers/JobRoleMapper.java

```diff
@@ -0,0 +1,32 @@
+package org.example.mappers;
+
+import org.example.models.JobRole;
+import org.example.models.JobRoleResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class JobRoleMapper {
+
+    private JobRoleMapper() {
+    }
+
+    public static JobRoleResponse toResponse(final JobRole jobRole) {
```

Comment:
should this method be private? And under the next one? Because the method below uses this one, we code from overal to specific